delivery/jwt.py

```diff
@@ -0,0 +1,192 @@
+import base64
+import collections.abc
+import dataclasses
+import datetime
+import enum
+import functools
+import logging
+
+import Crypto.PublicKey.RSA
+import Crypto.Util.number
+import dacite
+import jwt
+
+import model.delivery
+
+logger = logging.getLogger(__name__)
+JWT_KEY = 'bearer_token'
+
+
+class KeyType(enum.StrEnum):
+    RSA = 'RSA'
+    OCTET_SEQUENCE = 'oct' # used to represent symmetric keys
```

Comment:
Considering this (+ subsequent) enums/dataclasses are implemented according to an existing spec, it would make sense to add a module-level docstr stating this